README.md

```diff
@@ -25,84 +19,168 @@
+2. Reading our credentials from environment variables
+    * if available we can use environment variables **EDL_USERNAME** and **EDL_PASSWORD**
+3. Interactively entering our credentials
+    * We can ask for these credentials and even persist them to a `.netrc` file
 
 ```python
-from earthdata import Auth, DataGranules, DataCollections, Store
+import earthaccess
 
-auth = Auth().login(strategy="netrc") # if we want to access NASA DATA in the cloud
+auth = earthaccess.login(strategy="netrc")
+if not auth:
+    auth = earthaccess.login(strategy="interactive", persist=True)
+ ```
 
-# To search for collecrtions (datasets)
+Once we are authenticated with NASA EDL we can:
 
-DatasetQuery = DataCollections().keyword('MODIS').bounding_box(-26.85,62.65,-11.86,67.08)
+* Get a file from a DAAC using a `fsspec` session.
+* Requests temporary S3 credentials from a particular DAAC.
+* Use the library to download or stream data directly from S3.
+* Regenerate CMR tokens (used for restricted datasets)
 
-counts = DatasetQuery.hits()
-collections = DatasetQuery.get()
 
+## Searching for data
 
-# To search for granules (data files)
-GranuleQuery = DataGranules().concept_id('C1711961296-LPCLOUD').bounding_box(-10,20,10,50)
+Once we have selected our dataset we can search for the data granules using *doi*, *short_name* or *concept_id*.
+If are not sure or we don't know ohw to search for a particular dataset we can start with the "searching for data" tutorial or through the data pathfinder program listed above. For a complete list of search parameters we can use visit the API extended documentation.
 
-# number of granules (data files) that matched our criteria
-counts = GranuleQuery.hits()
-# We get the metadata
-granules = GranuleQuery.get(10)
+```python
 
-# earthdata provides some convenience functions for each data granule
-data_links = [granule.data_links(access="direct") for granule in granules]
+query = earthaccess.search_data(
+    short_name='ATL06',
+    version="005",
+    cloud_hosted=True,
+    bounding_box=(-10, 20, 10, 50),
+    temporal=("2020-02", "2020-03")
+)
+
+print(f"Granules found: {query.hits()}")
+
+# We execute our query and ask for 10 results only
+results = query.get(10)
+
+# If we want all the results we can iterate over items() 
+# for results in query.items():
+#     print(len(results))
+#     some cool stuff
+
+```
+
+Now that we have our results we can do multiple things, we can iterate over them to get HTTP (or S3) links; we can download the files to a local folder or we can open these files and stream their content directly to other libraries e.g. xarray.
+
+## Accessing the data
+
+### Option 1: we only want the links
+
+If we already have a workflow in place for downloading our data, we can use *earthaccess* as a search-only library and get HTTP links from our query results. This could be the case if our current workflow uses a different language and we only need the links as input.
```

Comment:
```suggestion
If you already have a workflow in place for downloading data, you can use *earthaccess* as a search-only library and get the HTTPS links from the query results. This could be useful if your current workflow uses a different language and you only need the links as input.
```